chart/config/releases.yaml

```diff
@@ -29,7 +29,7 @@ releases:
     condition: false
   cf-acceptance-tests:
     condition: testing.cf_acceptance_tests.enabled
-    version: 0.0.16
```

Comment:
Is this all it takes to consume this release? https://ci.flintstone.cf.cloud.ibm.com/teams/quarks/pipelines/release-images/jobs/build-cf-acceptance-tests/builds/53

Reply:
from https://github.com/cloudfoundry-incubator/kubecf/blob/master/doc/cf_deployment/bump.md#bump-cats-release

it seems to be it. The outdated docs point to when the versions were pinned in values.yaml, which I think informs us that there is no other place to change besides what you already changed.

Reply:
This looks correct to me (as long as there is an image of that version that has been built with the required stemcell).